llm4pddl/flags.py

```diff
@@ -13,7 +13,8 @@ def parse_flags() -> None:
     parser.add_argument("--approach", required=True, type=str)
     parser.add_argument("--seed", required=True, type=int)
     parser.add_argument("--experiment_id", default="", type=str)
-    parser.add_argument("--num_train_tasks", default=2, type=int)
+    parser.add_argument("--num_train_tasks", default=10, type=int)
```

Comment:
one consequence that we should be aware of is that all of the evaluation tasks are now going to be harder because we're shifting the offset. for that reason, can we actually make the default here 2 until we verify better performance with dynamic prompting vs what we already have?